### PR TITLE
shalev is idiot and his ice cream fell on the floor

### PR DESCRIPTION
**Pull request by ley3103**

### CODE REVIEW

File: yaml/ron/yaml_exercise1_ron.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+friends:
+   - name: Gal
+     age: 20
+     isPerson: false
+     isOlder: true
+
+# object / map
+family:
+  brothers:
+    ron:
```
Comment (etayruben) [inline yaml/ron/yaml_exercise1_ron.yaml:18]: you are not part of your brothers

File: yaml/ron/yaml_exercise1_ron.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        p.s kendrick lamar is the goat
+      series: [the wire,sopranos,avater]
+  sisters:
+    rosa:
+      kids: null
+      age: 28
+      isOlder: true
+      dateOfBirth: 14/7/1992
+    michal:
+      kids: null
```
Comment (etayruben) [inline yaml/ron/yaml_exercise1_ron.yaml:27]: Not true, she is pregnant

**Review by etayruben — DISMISSED** [review]

**Review by omershilohpsi — APPROVED** [review]

File: yaml/ron/yaml_exercise1_ron.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+# object / map
+family:
+  brothers:
+    ron:
+      kids: null
+      age: 19
+      isOlder: null
+      dateOfBirth: 31/3/2002
+      summery: >
```
Comment (benedery) [inline yaml/ron/yaml_exercise1_ron.yaml]: fix typo

File: yaml/ron/yaml_exercise1_ron.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        I dont know what to write so I write some bullshit
+        p.s kendrick lamar is the goat
```
Comment (benedery) [inline yaml/ron/yaml_exercise1_ron.yaml:16]: summary should be outside of "ron"

**Review by benedery — CHANGES_REQUESTED** [review]

**Pull request merged**